assets/src/edit-story/components/canvas/useFocusCanvas.js

```diff
@@ -22,8 +22,12 @@ import { useCallback } from 'react';
 function useFocusCanvas() {
   const focusCanvas = useCallback(() => {
     setTimeout(() => {
+      const doc = window.document;
+      if (doc.activeElement && doc.activeElement !== doc.body) {
+        doc.activeElement.blur();
+      }
```

Comment:
If this happens, then we don't need the fake event, do we? Shouldn't we just return here?

Reply:
I still want the blur event, at least when the active element is the body.